buildbuddy.yaml

```diff
@@ -23,7 +23,7 @@ x_templates:
         USE_BAZEL_VERSION: 7.x
     - &bazel_head
       env:
-        USE_BAZEL_VERSION: last_green
+        USE_BAZEL_VERSION: dd2464a5933e0a5a6765024573832717b71989bf
```

Comment:
Please add a comment above this.

Reply:
https://github.com/MobileNativeFoundation/rules_xcodeproj/pull/3029/commits/066425f95b82157282536cea6b3524320cd17f67